backend/lib/endpoints/posts.js

```diff
@@ -104,20 +104,18 @@ async function routes(app) {
 
       // Unlogged user limitation for post content size
       /* eslint-disable sort-keys */
-      const contentProjection = user
-        ? "$content"
-        : {
```

Comment:
Removing this conditional will cause all posts to be capped to 150 char even if user is logged. Is it intentional?

Reply:
So this is the post content that's retrieved in the feed. There is no toggling for more content in the feed, this now happens in the post page in the `posts/{postID}` route. I just need the first 120 characters of the post's content for every post in the feed for everyone, logged in or not. This is the intended behavior.

Reply:
Ok, this seems okay!